docs/install/install-welcome.md

```diff
@@ -5,7 +5,7 @@
 
 OpenCAD Installation Requirements include the following: -* Operating System: Linux or Windows
 
- - Webserver: Apache or Nginx
+ - Webserver: Apache, Nginx, or Internet Information Services
  - PHP: 7.3 (Recommended), though 7.2 is the minimum
  - Database: MySQL ≥ 5.5 or MariaDB ≥ 10.0, though we recommend 10.3
-The application can be installed on shared hosting environments.Please note that **FREE** shared hosting not reccomended as the quality of service, in the organization's opinion, leave a lot to be desired.
\ No newline at end of file
+The application can be installed on shared hosting environments. Please note that **FREE** shared hosting is not recommended as the quality of service, in the organization's opinion, leaves a lot to be desired.  
```

Comment:
I believe in UK English this was actually correct. "leave" vs. "leaves". I sort of default to UK English.

We really haven't specified weather the project is UK or US English. We have identified that is primarily English.